java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -258,8 +258,8 @@ public Result getDriverPath(Capabilities options, boolean offline) {
     List<String> arguments = new ArrayList<>();
     arguments.add("--browser");
     arguments.add(options.getBrowserName());
-    arguments.add("--language-binding");
-    arguments.add("java");
```

Comment:
This should work well if you update the branch.

Reply:
Yes, thank you!